sig-robotics/proposal/Definition of cloud robot CRD and Operator/Definition of cloud robot CRD and Operator.md

```diff
@@ -0,0 +1,157 @@
+# Definition of Cloud Robot CRD and Operator
+
+## Background
+
+The emergence of cloud-native architecture has further facilitated the development of cloud-native robotics technology. Cloud-native architecture encompasses characteristics such as containerization, microservices, and elastic scalability. These features enable robotic systems to adapt more effectively to complex work environments and diverse task requirements. Leveraging containerization technology for deployment and management, the maintainability, scalability, and flexibility of robot systems are significantly improved.
+
+Despite the widespread application of cloud-native robotics in industries such as industrial automation and logistics and warehousing, there is still a lack of a universal management and control solution that offers high availability and scalability for robots. This significantly hinders the broader adoption of cloud-native robotics and impedes further development in the robotics industry. A comprehensive set of CRDs and Operators is thus needed to enhance the scalability, flexibility, and availability of robots.
+
+This proposal presents a design solution for a universal CRDs and Operator, enabling resource monitoring and automated control of cloud-native robots. 
+
+## Goals
+
+For developers or end users of CRDs and Operators, the goal of this project is to provide the following capabilities
+
+CRDs are defined to achieve
+
+- Difining robot resources
+- Extending the robot API to enable robot control
+
+Operator is designed to achieve
+
+- automatizing  robotic operations
+- Monitoring robot status
+- automatizing fault handling
+- adaptive controlling
+
+## Proposal
+
+The CRDs and Controllers is designed to provide more flexible and scalable management solutions for cloud robots connected to the KubeEdge platform.
+
+project scope：
+
+The scope of this project includes
+
+- Providing a series of CRDs  and an Operator for managing cloud robotics based on KubeEdge architecture
+  - CRDs to define robot properties and states
+  - CRDs to extend teh robot API, through which we can control the robot
+  - A series of controllers to realize the automatic control of the robot
+- This project is to design the robot CRD based on the interface definition of industrial mobile robots proposed by the Mobile Robot Industry Alliance
```

Comment:
**Suggested Addition**: Adding Reference Link for "the interface definition of industrial mobile robots proposed by the Mobile Robot Industry Alliance"

To provide more context and allow interested readers to explore further details, it is recommended to include a reference link to "the interface definition of industrial mobile robots proposed by the Mobile Robot Industry Alliance." This will enable readers to access the original source and gain a deeper understanding of the specific interface definitions used in the proposed CRD design.

Reply:
done